### DISCUSSION TITLE
Build bug, when resizeble true enabled

### DISCUSSION THREAD

**Discussion by AyubCompetitor** [Comcast/react-data-grid#3464] [Q&A]
Hi, we are using react-data-grid on our project. 
I have one strange problem when using the grid, which is as follows, when resizeble is enabled and the build is being built, the column layout is broken, although this is not observed when starting via npm start, has anyone encountered this? I attach screenshots below. 
![SCR](https://github.com/adazzle/react-data-grid/assets/100413597/49f9bdf9-0c2e-48e1-ac4a-555224c9826c)

**Comment by FoxCDD** [comment]
Yes, we've encountered this as well. I've raised it as a bug (#3463), waiting for a response. At the moment we're setting resizable to false and setting the width accordingly so we can see the header.
